doc/config.md

```diff
@@ -38,6 +40,7 @@ can lead to serious problems! Currently the *only* permitted operation is *delet
 The Toml-formatted config files ([an example here](config_example.toml)) are (from lowest priority to highest): `/etc/electrs/config.toml`, `~/.electrs/config.toml`, `./electrs.toml`.
 
 The options in highest-priority config files override options set in lowest-priority config files.
+In case even an attempt to load these files is undesirable (common in case of protected systemd services) use the `--skip-default-conf-files` argument to prevent it.
```

Comment:
```suggestion
In case when an attempt to load these files is undesirable (common in case of protected systemd services), use the `--skip-default-conf-files` argument to prevent it.
```

Reply:
Got it, thanks!